src/usr/share/gpgmailer/gpgmailmessage.py

```diff
@@ -18,24 +18,25 @@
 
 __all__ = [
     'WatchDirectoryMissingException', 'SaveMessageWithoutBodyException',
-    'ModifyAlreadySavedMessageException', 'GpgMailMessageNotConfiguredException',
-    'GpgMailMessage']
+    'ModifyAlreadySavedMessageException', 'GpgMailMessage']
 __author__ = 'Joel Luellwitz, Andrew Klapp, and Brittney Scaccia'
 __version__ = '0.8'
 
-import confighelper
-import ConfigParser
 import base64
 import datetime
-import hashlib
 import json
 import logging
 import os
 import shutil
+import stat
+
+SPOOL_DIR = '/var/spool/gpgmailer'
+PARTIAL_DIR = os.path.join(SPOOL_DIR, 'partial')
+OUTBOX_DIR = os.path.join(SPOOL_DIR, 'outbox')
 
 
 class WatchDirectoryMissingException(Exception):
-    """This exception is raised when gpgmailer is configured but the watch directories do not
+    """This exception is raised when gpgmailer is configured but the spool directories do not
```

Comment:
spool directories **are** not, or spool directories do not **exist**. :stuck_out_tongue_closed_eyes:

Reply:
It is on the next line.